### PR TITLE
Update PE/CM and simplify err logging

### PR DESCRIPTION
**Pull request by decentralgabe**

### CODE REVIEW

File: pkg/server/router/manifest.go [hunk 1]
```diff
@@ -41,6 +41,8 @@ func NewManifestRouter(s svcframework.Service) (*ManifestRouter, error) {
 // CreateManifestRequest is the request body for creating a manifest, which populates all remaining fields
 // and builds a well-formed manifest object.
 type CreateManifestRequest struct {
+	ManifestName           *string                          `json:"name,omitempty"`
+	ManifestDescription    *string                          `json:"description,omitempty"`
```
Comment (andresuribe87) [inline pkg/server/router/manifest.go:45]: Consider this as the manifest bit is implied. 
```suggestion
	Name           *string                          `json:"name,omitempty"`
	Description    *string                          `json:"description,omitempty"`
```

**Review by andresuribe87 — APPROVED** [review]
Looks good. Tiny comment.

File: pkg/server/router/manifest.go [hunk 2]
```diff
@@ -50,6 +52,8 @@ type CreateManifestRequest struct {
 
 func (c CreateManifestRequest) ToServiceRequest() manifest.CreateManifestRequest {
 	return manifest.CreateManifestRequest{
+		ManifestName:           c.ManifestName,
+		ManifestDescription:    c.ManifestDescription,
```
Comment (decentralgabe) [inline pkg/server/router/manifest.go:56]:
```suggestion
		Name:           c.Name,
		Description:    c.Description,
```

File: pkg/service/manifest/model.go [hunk 3]
```diff
@@ -11,6 +11,8 @@ import (
 // Manifest
 
 type CreateManifestRequest struct {
+	ManifestName           *string                          `json:"name,omitempty"`
+	ManifestDescription    *string                          `json:"description,omitempty"`
```
Comment (decentralgabe) [inline pkg/service/manifest/model.go:15]:
```suggestion
	Name           *string                          `json:"name,omitempty"`
	Description    *string                          `json:"description,omitempty"`
```

File: pkg/service/manifest/service.go [hunk 4]
```diff
@@ -86,13 +85,24 @@ func (s Service) CreateManifest(request CreateManifestRequest) (*CreateManifestR
...
+		if err := builder.SetName(*request.ManifestName); err != nil {
+			return nil, util.LoggingErrorMsg(err, "invalid manifest name")
+		}
+	}
+	if request.ManifestDescription != nil {
+		if err := builder.SetDescription(*request.ManifestDescription); err != nil {
+			return nil, util.LoggingErrorMsg(err, "invalid manifest description")
+		}
+	}
+
```
Comment (decentralgabe) [inline pkg/service/manifest/service.go:105]:
```suggestion
	if request.Name != nil {
		if err := builder.SetName(*request.Name); err != nil {
			return nil, util.LoggingErrorMsg(err, "invalid manifest name")
		}
	}
	if request.Description != nil {
		if err := builder.SetDescription(*request.Description); err != nil {
			return nil, util.LoggingErrorMsg(err, "invalid manifest description")
		}
	}

```

**Pull request merged**